slirp4netns.1.md

```diff
@@ -207,14 +213,22 @@ slirp4netns can provide QMP-like API server over an UNIX socket file:
 (host)$ slirp4netns --api-socket /tmp/slirp4netns.sock ...
 ```
 
-`add_hostfwd`: Expose a port (IPv4 only)
+`add_hostfwd`: Expose a port (IPv4 and IPv6 if enabled and host_addr is "0.0.0.0" or "::")
```

Comment:
What do you mean?

Reply:
I will try to write it better, i mean this https://github.com/AkihiroSuda/rootlesskit/blob/5da949974a6e333902799c881ba0c446858723cb/docs/port.md